Update scRNA-seq introduction notebook (#480)
AlexsLemonade/training-modules: scRNA-seq/00-scRNA_introduction.md

(This will result in two fastq files if this is paired-end sequencing.)
The data preprocessing steps for these types of scRNA-seq data are similar to bulk RNA-seq methods.
For the purposes of this tutorial, we will focus on tag-based scRNA-seq, but it is important to keep in mind that the pre-processing steps and the biases to look out for in post-processing vary based on technology and how the cells are sorted.

#### Pros:  
- Can be paired-end sequencing which has less risk for 3' bias.  
- More complete coverage of transcripts, which may be better for transcript discovery purposes.   
For more extensive background on single-cell experimental methods, Kiselev _et al._ have a very good [tutorial for scRNA-seq](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html).

#### Cons:  
- Is not very efficient (generally 96 cells per plate).  
- Takes much longer to run (days/weeks depending on sample size).
- Much more expensive.  
![**Overall view of scRNA-seq tag-based workflow**](diagrams/overview_workflow.png)

### Tag-based scRNA-seq  
*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)    

> Cells are separated by emulsion/droplets, and individual cells are given barcodes.

Reviewer: This point seems important to keep! I think it might be worth laying out with even a bit more detail than we had, but the separation step should be there, and the fact that there are (at least) two barcodes

> Individual samples have two fastq files: one for the cell barcodes (R1) and another with the individual reads (R2).

Reviewer: Some clarification?
```suggestion
Individual samples have two fastq files: one for the cell and UMI barcodes (R1) and another with the transcript seqeunce reads (R2).
```

In *full-length* scRNA-seq, cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
With full-length scRNA-seq, each cell is sequenced individually and has its own fastq file. 
An example of full-length scRNA-seq is Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006).

For the purposes of this tutorial, we will focus on tag-based scRNA-seq, but it is important to keep in mind that the pre-processing steps and the biases to look out for in post-processing vary based on technology and how the cells are sorted.

For more extensive background on single-cell experimental methods, Kiselev _et al._ have a very good [tutorial for scRNA-seq](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html).

![**Overall view of scRNA-seq tag-based workflow**](diagrams/overview_workflow.png)

### Tag-based scRNA-seq  
*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)
Individual cells are separated by emulsion/droplets prior to cell lysis. 

> Transcripts from each cell are then tagged with two barcodes: a unique cell barcode and a Unique Molecular Identifier (UMI) 

Reviewer: Changing wording to reduce ironic redundancy of "unique". 

I also added the reference for UMIs here, though I guess that could be confusing given the fact that UMIs are not necessarily specific to this method. But I think it is okay.
 
```suggestion
Transcripts from each cell are then tagged with two barcodes: a cell-specific barcode and a Unique Molecular Identifier (UMI) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772). 
```

All transcripts from all cells are then pooled together and undergo PCR amplification and sequencing as if they are one sample.

> These types of methods, because they are newer, are more likely to have (UMIs) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772). 

Reviewer: I think this is now redundant?
```suggestion
```

#### Cons:  
- Sequencing is not bidirectional so data will likely have more intense 3' bias.  

> - Coverage of these technologies is generally not as deep. 

Reviewer: I thought the wording here could be clarified. 
```suggestion
- The sequencing depth per cell with these technologies is generally lower.
```
